spec/models/form526_submission_spec.rb

```diff
@@ -936,4 +936,44 @@
       end
     end
   end
+
+  describe '#disabilities_not_service_connected?' do
+    subject { form_526_submission.disabilities_not_service_connected? }
+
+    let(:form_526_submission) do
+      Form526Submission.create(
+        user_uuid: user.uuid,
+        saved_claim_id: saved_claim.id,
+        auth_headers_json: auth_headers.to_json,
+        form_json: File.read("spec/support/disability_compensation_form/submissions/#{form_json_filename}")
+      )
+    end
+
+    before { VCR.insert_cassette('evss/disability_compensation_form/rated_disabilities_with_non_service_connected') }
+    after { VCR.eject_cassette('evss/disability_compensation_form/rated_disabilities_with_non_service_connected') }
```

Comment:
I haven't use these VCR methods, are they an alternative to wrapping the `context`s in a VCR block?

Reply:
That's plausible -- I didn't realize entire (and multiple?) `context` blocks could be nested within a VCR block.